src/Analyzers/Core/Analyzers/EnforceOnBuildValues.cs

```diff
@@ -75,7 +75,8 @@ internal static class EnforceOnBuildValues
         public const EnforceOnBuild RemoveUnnecessaryDiscardDesignation = /*IDE0110*/ EnforceOnBuild.Recommended;
         public const EnforceOnBuild InvokeDelegateWithConditionalAccess = /*IDE1005*/ EnforceOnBuild.Recommended;
         public const EnforceOnBuild NamingRule = /*IDE1006*/ EnforceOnBuild.Recommended;
-        public const EnforceOnBuild MatchFolderAndNamespace = /* IDE0130*/ EnforceOnBuild.Recommended;
+        public const EnforceOnBuild MatchFolderAndNamespace = /*IDE0130*/ EnforceOnBuild.Recommended;
+        public const EnforceOnBuild SimplifyObjectCreation = /*IDE0131*/ EnforceOnBuild.Recommended;
```

Comment:
can you move this to 140.  i want more space for Folder-Sync for more diag ids.

Reply:
@CyrusNajmabadi The id is actually 140, but the comment is outdated. Will fix.